index.html

```diff
@@ -1979,139 +1962,29 @@ <h2>The <dfn>record type</dfn> string</h2>
   </section>
 </section> <!-- Data types and content -->
 
-
-<section> <h2>The NDEFReader and NDEFWriter objects</h2>
-  The objects provide a way for the <a>browsing context</a> to
-  use NFC functionality.
-  They allow for writing <a>NDEF message</a>s to <a>NFC tag</a>s within range,
-  and to act on incoming <a>NDEF message</a>s from <a>NFC tag</a>s.
+<section> <h2>Extensions to the Navigator interface</h2>
   <pre class="idl">
-    typedef (DOMString or BufferSource or NDEFMessageInit) NDEFMessageSource;
-
-    [SecureContext, Exposed=Window]
-    interface NDEFWriter {
-      constructor();
-
-      Promise&lt;undefined&gt; write(NDEFMessageSource message,
-                                     optional NDEFWriteOptions options={});
-    };
-
-    [SecureContext, Exposed=Window]
-    interface NDEFReader : EventTarget {
-      constructor();
-
-      attribute EventHandler onerror;
-      attribute EventHandler onreading;
-
-      Promise&lt;undefined&gt; scan(optional NDEFScanOptions options={});
+    [SecureContext]
+    partial interface Navigator {
+      [SameObject]
+      readonly attribute NFC nfc;
     };
+  </pre>
+</section>
 
+<section data-dfn-for="NFC">
+<h2>The NFC object</h2>
+  <pre class="idl">
     [SecureContext, Exposed=Window]
-    interface NDEFReadingEvent : Event {
-      constructor(DOMString type, NDEFReadingEventInit readingEventInitDict);
-
-      readonly attribute DOMString serialNumber;
-      [SameObject] readonly attribute NDEFMessage message;
-    };
-
-    dictionary NDEFReadingEventInit : EventInit {
-      DOMString? serialNumber = "";
-      required NDEFMessageInit message;
+    interface NFC {
+      readonly attribute NDEFReader ndef;
     };
   </pre>
-  <p>
-    The <dfn>NDEFMessageSource</dfn> is a union type representing argument types
-    accepted by the {{NDEFWriter/write()}} method.
+  <p>The
+    <dfn>ndef</dfn> property is an {{NDEFReader}} object that provides
+    <a>NDEF</a> tag read and write functionality.
```

Comment:
Shall we use NFC tags consistently?

Reply:
True, changed. I was thinking not all NFC tags were NDEF, so using "NDEF tag" was more appropriate for NDEFReader.